heron/schedulers/tests/java/com/twitter/heron/scheduler/local/LocalSchedulerTest.java

```diff
@@ -159,7 +169,7 @@ public void testRemoveContainer() throws Exception {
     containersToRemove.add(containerToRemove);
     scheduler.removeContainers(containersToRemove);
     existingContainers.remove(containerToRemove);
-    Assert.assertEquals(existingContainers.size(), scheduler.getProcessToContainer().size());
+    Assert.assertEquals(1 + existingContainers.size(), scheduler.getProcessToContainer().size());
```

Comment:
If we're not already doing so, we should assert proper container ids as well as total container count.